home/templates/home/signup.html

```diff
@@ -0,0 +1,12 @@
+{% extends "home/base.html" %}
+
+{% block content %}
+<body class="main-bg">
```

Comment:
@idokk1
I recommend that you also add margins so that this screen will look better and the form won't be too close to the sides.

Reply:
fixed at login page, here it's on purpose. there is a lot of text (like the minimum length of password ) and when it's on the center it's look worst